libs/core/coroutines/include/hpx/coroutines/detail/get_stack_pointer.hpp

```diff
@@ -8,6 +8,7 @@
 
 #pragma once
 
+#include <hpx/config.hpp>
 #include <hpx/config/compiler_specific.hpp>
 #include <hpx/config/defines.hpp>
```

Comment:
If you include <hpx/config.hpp>, then the other two #includes are not needed anymore (those are pulled in by it).